src/client/components/Post/index.js

```diff
@@ -5,15 +5,37 @@ import PostInfo from '../PostInfo/';
 import PostAct from '../PostAct/';
 
 class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      numOfLikes: this.props.item.numOfLikes,
+    }
+  }
+  like(event) {
+    let xhr = new XMLHttpRequest();
+    let token = window.localStorage.getItem('token');
+
+    xhr.addEventListener('readystatechange', function() {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
+        this.setState({numOfLikes: xhr.response})
```

Comment:
please add some basic error handling, if something goes wrong the js will break...